std/node/_fs/_fs_lstat.ts

```diff
@@ -36,9 +36,10 @@ export function lstat(
 
   if (!callback) throw new Error("No callback function supplied");
 
-  Deno.lstat(path)
-    .then((stat) => callback(undefined, CFISBIS(stat, options.bigint)))
-    .catch((err) => callback(err, err));
+  Deno.lstat(path).then(
+    (stat) => callback(undefined, CFISBIS(stat, options.bigint)),
+    (err) => callback(err, err),
```

Comment:
```suggestion
    (err) => callback(err),
```
(`arguments.length` == 1 when a fs callback fails)

Similar comments apply to `std/node/_fs/_fs_open.ts` and `std/node/_fs/_fs_stat.ts`.